pkg/controller/stacks/request/stackrequest.go

```diff
@@ -60,6 +61,11 @@ var (
 	resultRequeue    = reconcile.Result{Requeue: true}
 	requeueOnSuccess = reconcile.Result{RequeueAfter: requeueAfterOnSuccess}
 	jobBackoff       = int32(0)
+
+	// podImageNameEnvVar is the env variable for setting the image name used for the extension manager unpack/install process.
+	// When this env variable is not set the parent Pod will be detected and the associated image will be used.
+	// Overriding this variable is only useful when debugging the main extension manager process, since there is no Pod to detect.
+	podImageNameEnvVar = "EXTENSION_MANAGER_IMAGE"
```

Comment:
want to update the lingering instances of "extension" to "stack"?

Reply:
Stacked it away.